ui/app/routes/vault/cluster/clients/counts.ts

```diff
@@ -49,7 +58,10 @@ export default class ClientsCountsRoute extends Route {
     mountPath: { refreshModel: false, replace: true },
   };
 
-  async getActivity(start_time: number, end_time: number) {
+  async getActivity(
+    start_time: number,
+    end_time: number
+  ): Promise<[ClientsActivityModel | undefined, AdapterError | unknown] | [Record<string, never>, null]> {
```

Comment:
Why was updating this typescript declaration necessary?

Reply:
this happened as a result of converting this file to typescript. without these types explicitly defined here, TypeScript wasn't able to correctly infer what the return type was. as a result:

```
const [activity, activityError] = await this.getActivity(startTimestamp, endTimestamp);
// ^ typescript thinks activity is of type 'unknown'

const isSecretsSyncActivated = await this.isSecretsSyncActivated(activity);
// ^ passing activity with a type of 'unknown' causes a type error bc isSecretsSyncActivated can't handle 'unknown'
```

i tried a few things but i figured the best 2 options were to either:
1. correctly define the return type, as i've done here. it's ugly but it _is_ correct, since we _are_ confident that this function is going to always return either the activity response or `{}`
2. make `isSecretsSyncActivated(activity: any)`. this also works fine but it felt Not Right since we know what `getActivity` is going to return, it's just ugly 😬